docs/source/markdown/podman-machine-init.1.md

```diff
@@ -76,15 +76,32 @@ Set the timezone for the machine and containers.  Valid values are `local` or
 a `timezone` such as `America/Chicago`.  A value of `local`, which is the default,
 means to use the timezone of the machine host.
 
-#### **--volume**, **-v**=*source:target*
+#### **--volume**, **-v**=*source:target[:options]*
 
 Mounts a volume from source to target.
 
 Create a mount. If /host-dir:/machine-dir is specified as the `*source:target*`,
 Podman mounts _host-dir_ in the host to _machine-dir_ in the Podman machine.
 
-The root filesystem is mounted read-only in the default operating system,
-so mounts must be created under the /mnt directory.
+Additional options may be specified as a comma-separated string. Recognized
+options are:
+* **ro**: mount volume read-only
+* **rw**: mount volume read/write (default)
+* **security_model=[model]**: specify 9p security model (see below)
```

Comment:
If we swap out 9p for virtiofs in the future, should we just document this as the security model flag, in case there were other file systems security model?

Reply:
Possibly.

I haven't used virtiofs yet, though, and I was unable to find documentation of any feature that corresponds to 9p's security model.

If somebody knows better, than I'll change that phrasing, but otherwise, I think it best for the wording to be specific and not give the impression that the option does anything more generic.